src/main/resources/application.yml

```diff
@@ -0,0 +1,19 @@
+spring:
+  cloud:
+    gateway:
+      routes:
+      - id: catalog_route
+        uri: http://localhost:8081
```

Comment:
It is possible to use following syntax to inject real url in runtime, and specify default in the same line
${ANY_URL:http://localhost:8081}/books

It is useful in case URL is not a static value, like when Kubernetes is used.